code/datums/supplypacks/supply.dm

```diff
@@ -114,14 +114,14 @@
 	containertype = /obj/structure/closet/crate/secure
 	containername = "Shaft miner equipment"
 	access = access_mining
-/* //VOREStation Edit - Pointless on Tether.
+
```

Comment:
For now, it'd be nice to keep this commented out, since MULEbots legitimately can't function on Tether

Reply:
We will eventually be switching maps, and although they're pointless for now, they won't be in the future. It can always be commented back out after this is merged.